src/components/item/ItemTags.vue

```diff
@@ -50,13 +58,25 @@ export default {
       'addTagToListLocally',
     ]),
     ...mapActions('items', [
-      '_addItemOnServer',
-      '_updateItemOnServer',
       '_saveItemOnServer',
     ]),
     ...mapActions('lists', [
       '_addTagForListAndItem',
     ]),
+    checkTitleUniqueness(categoryTitle) {
+      const isTitleUnique = !this.currentListCategoriesTitles.includes(categoryTitle)
+        && !this.currentListTagsTitles.includes(categoryTitle);
+
+      this.requestHandling.errorMessage = isTitleUnique
+        ? ''
+        : 'filter with this name already exists';
+
+      this.newTitle = isTitleUnique
+        ? categoryTitle
+        : '';
+
+      return isTitleUnique;
+    },
```

Comment:
it's repeating code.
let's think about how we can handle it for making universal